OMPython/__init__.py

```diff
@@ -447,3 +462,727 @@ def getClassNames(self, className=None, recursive=False, qualified=False, sort=F
+		else:
+
+			exp = '{}()'.format(apiName)
+		try:
+			res = self.getconn.sendExpression(exp)
+		except pyparsing.ParseException as e:
+			res = 'Message: Fault'
+		return res
+		
+		
+	def requestApi__(self, apiName, entity ):
+			
+		exp = '{}({})'.format(apiName, entity)
+		
+		res = self.getconn.sendExpression(exp)
+			
+		return res
+
+	def createQuantitiesList(self):
+		rootCQ = self.root
+		if self.quantitiesList == []:
+			for sv in rootCQ.iter('ScalarVariable'):
+				name = sv.get('name')
+				changable = sv.get('isValueChangeable')
+				description = sv.get('description')
+				variability = sv.get('variability')
+				causality = sv.get('causality')
+				ch = sv.getchildren()
+				start = None
+				for att in ch:
+					start = att.get('start')
+				self.quantitiesList.append(Quantity(name, start, changable, variability, description, causality))
+		return self.quantitiesList
+		
+	def getQuantitiesNames(self):
+		if self.qNamesList == []:
+			for q in self.quantitiesList:
+				self.qNamesList.append(q.name)
+		else:
+			pass
+		return self.qNamesList
+		
+	def checkAvailability(self, names, chkList, inputFlag = None):
+		isExist = True
+		if inputFlag is not None:
+			if names in chkList:
+				pass
+			else:
+				isExist = False
+			if(isExist == False):
+				print 'Error!!! ', names , ' does not exist.'
+				return False
+		
+		else:
+			nonExistingList = []
+			for n in names:
+				if n in chkList:
+					pass
+				else:
+					isExist = False
+					nonExistingList.append(n)
+
+		if(isExist == False):
+			print 'Error!!! ', nonExistingList , ' does not exist.'
+			return False
+		else:
+			return True
+	
+	def getQuantities(self, names = None):
+		str_ = ''
+		strList = []
+		if (names == None):
+			nameList = []
+			startList =[]
+			changableList = []
+			variabilityList = []
+			descriptionList = []
+			
+			for q in self.quantitiesList:
+				nameList.append(q.name)
+				startList.append(q.start)
+				changableList.append(q.changable)
+				variabilityList.append(q.variability)
+				descriptionList.append(q.description)
+
+			dic = {'Name' : nameList, 'Value' : startList, 'Changable' : changableList, 'Variability' : variabilityList, 'Description' : descriptionList}
+			frame = pd.DataFrame(dic)
+
+		else:
+			try:
+				checking = self.checkAvailability(names, self.qNamesList)
+				if(checking is False):
+					return None
+				
+				nameList = []
+				startList =[]
+				changableList = []
+				variabilityList = []
+				descriptionList = []
+				for n in names:
+					for q in self.quantitiesList:	
+						if n ==q.name:
+							nameList.append(q.name)
+							startList.append(q.start)
+							changableList.append(q.changable)
+							variabilityList.append(q.variability)
+							descriptionList.append(q.description)
+
+				dic = {'Name' : nameList, 'Value' : startList, 'Changable' : changableList, 'Variability' : variabilityList, 'Description' : descriptionList}	
+				frame = pd.DataFrame(dic)
+			except Exception as e:
+				return e
```

Comment:
probably at least a `print e` should be added here, instead of silently swallowing any exception.